docs/backend/SYCL.md

```diff
@@ -41,6 +41,8 @@ The following release is verified with good quality:
 
 ## News
 
+- 2024.11
+  - Use syclcompat to improve the performance on some backends. This requires to use oneAPI 2025.0 or more recent.
```

Comment:
"some backends" is strange to SYCL backend.
Maybe "some platforms" or “some GPUs".

"more recent" -> "newer"

Reply:
Reworded this in https://github.com/ggerganov/llama.cpp/pull/10267/commits/1c1651600485366cadb671747192640be5ee9bcd